pkg/disruptors/pod.go

```diff
@@ -136,17 +139,15 @@ func NewPodDisruptor(
 		return nil, fmt.Errorf("finding pods matching '%s': %w", selector, ErrSelectorNoPods)
 	}
 
-	controller := NewPodController(targets)
-
 	return &podDisruptor{
-		helper:     helper,
-		options:    options,
-		controller: controller,
+		helper:  helper,
+		options: options,
+		targets: targets,
 	}, nil
 }
 
-func (d *podDisruptor) Targets(ctx context.Context) ([]string, error) {
-	return d.controller.Targets(ctx)
+func (d *podDisruptor) Targets(_ context.Context) ([]string, error) {
+	return utils.PodNames(d.targets), nil
```

Comment:
Seems like the only two implementation of `Targets` no longer use a context neither return an error, should we update the interface to reflect this?

![image](https://github.com/grafana/xk6-disruptor/assets/969721/69b03b29-709a-4dbd-b800-a5ff57f8e67a)